.buildkite/run.sh

```diff
@@ -58,6 +58,8 @@ fi
 echo "--- Updating submodules"
 time git submodule update --init --recursive
 time git submodule foreach "git fetch --all"
+time git submodule foreach "git checkout -B dev -t origin/dev"
+time git submodule foreach "git checkout master"
```

Comment:
Will we need to do this for every new branch?
Should this also be added to the testing pre-requisites in DEVELOPMENT.md?

Reply:
Haven't found a good solution for yet, we want a certain state local branches. Any ideas how to make this more elegant?

Reply:
No idea at the moment.

Reply:
Created #207 as follow-up.